src/components/templates/SharedLayout/SharedLayout.jsx

```diff
@@ -1,16 +1,20 @@
 import { Outlet } from 'react-router-dom';
 
+import { About } from '../About/About.jsx';
 // import { Header } from '../Header/Header.jsx';
 // import { Footer } from '../Footer/Footer.jsx';
 // import { Container } from '../Container/Container.jsx';
 
 function SharedLayout() {
   return (
-    // <Container>
-    //   <Header />
-    <Outlet />
-    //   <Footer />
-    // </Container>
+    <>
+      {/*  <Container>
+     <Header /> */}
+      <About />
```

Comment:
If you want to use About component its need to be in includes as a outlet so don't put it inside Shared Layout component 💡